client/components/chip/index.tsx

```diff
@@ -0,0 +1,31 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export type chipType = 'primary' | 'light' | 'warning' | 'alert';
+
+const Chip = ( props: {
+	message: string;
+	type?: chipType;
+	isCompact?: boolean;
+} ): JSX.Element => {
```

Comment:
Just out of curiosity. Probably there's been some discussions about this but why don't we do the following:

```tsx
const Chip: React.FC<Props> = (props: Props) => {
```

is it overkill or do you want to be more specific about the returning value of the function.

Reply:
There's been an interesting discussion around this in P2 (paJDYF-167-p2#comment-5199) and [Slack](https://a8c.slack.com/archives/CLCNELZE1/p1614264592024900) actually! I think it might be best to rely on type inference here, but I'm not sure.

I'm going to switch to using type inference here until we settle on something else, since that should work a bit better during development.

Excellent question, thanks for bringing it up! 👏